src/Main.java

```diff
@@ -0,0 +1,38 @@
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        Integer[] numeric = {1,2,3,4,5,6,7,8,9,0};
+        String[] alphabetic = {"een", "twee", "drie", "vier", "vijf", "zes", "zeven", "acht","negen","nul"};
+
+        Translator myFirstHashmap = new Translator(alphabetic, numeric);
```

Comment:
probeer goede variable namen te verzinnen. Het Translator object dat je hier aan maakt is geen hashMap, maar een Translator object. 'translator' of desnoods 'myFirstTranslator' of 'numbericToAlpabeticTranslator' zijn beter variable namen